gubernator/main.py

```diff
@@ -34,11 +35,8 @@
 import filters
 
 BUCKET_WHITELIST = {
-    'kubernetes-jenkins',
-    'kube_azure_log',
-    'rktnetes-jenkins',
-    'kubernetes-github-redhat',
-    'kopeio-kubernetes-e2e',
+    re.match(r'gs://([^/]+)', path).group(1)
+    for path in yaml.load(open("buckets.yaml"))
```

Comment:
The tiny cost of pushing gubernator and the observed very low rate of changes to the list of buckets doesn't seem to necessitate a more complex solution.

We can make Travis deploy gubernator from master in the future if necessary.